packages/rsbuild-plugin-angular/src/lib/plugin/utils/source-file-cache.ts

```diff
@@ -34,7 +33,7 @@ export class SourceFileCache extends Map<string, ts.SourceFile> {
       this.typeScriptFileCache.delete(pathToFileURL(file).href);
 
       // Normalize separators to allow matching TypeScript Host paths
-      if (USING_WINDOWS) {
+      if (isUsingWindows()) {
```

Comment:
I extracted the const into a runtime check to be able to test `SourceFileCache` fully